### ISSUE TITLE
Brick\Math\Exception\NumberFormatException : "4" is not a valid character in base 2.

### ISSUE THREAD

**Issue by stephanvierkant** [web-token/jwt-framework#302]
https://github.com/web-token/jwt-util-ecc/commit/a7ec5beb004a171f0305f8ba771d8b0ee9823839 breaks [minishlink/web-push](https://github.com/web-push-libs/web-push-php) (using v6.0.3).

> Brick\Math\Exception\NumberFormatException : "4" is not a valid character in base 2.

Stacktrace:
```
vendor/brick/math/src/BigInteger.php:120
vendor/web-token/jwt-util-ecc/Point.php:131
vendor/web-token/jwt-util-ecc/Point.php:109
vendor/web-token/jwt-util-ecc/Curve.php:206
vendor/web-token/jwt-util-ecc/Curve.php:274
vendor/minishlink/web-push/src/Encryption.php:276
vendor/minishlink/web-push/src/Encryption.php:265
vendor/minishlink/web-push/src/Encryption.php:65
vendor/minishlink/web-push/src/WebPush.php:225
vendor/minishlink/web-push/src/WebPush.php:173
vendor/minishlink/web-push/src/WebPush.php:144
```

**Comment by stephanvierkant** [comment]
@Spomky I've no idea how to fix this, but please let me know if I can help. Does a reproducible example help? Or maybe you can point me in the right direction so I can try to fix this?

**Comment by Spomky** [comment]
Hi,

This should be fixed with the new ``v2.2.10`.

I take the opportunity to tell you that I started working on [web-push](https://github.com/Spomky-Labs/web-push). The project is not yet marked as stable, but I have not much changes before the first release.
This new project is based on the work I proposed in the PR https://github.com/web-push-libs/web-push-php/pull/302.
You may be interested in this project (for testing purpose).

Regards.

**Comment by stephanvierkant** [comment]
Thanks! I'm using PHP 8 and v2.2.10 works for me.
